obspy/clients/eida/client.py

```
@@ -0,0 +1,140 @@
+    def __init__(self, base_url='GFZ', retry_count=10, retry_wait=60,
+                 maxthreads=5, credentials=None, authdata=None, **kwargs):
+        """
+        Initializes an FDSN/EIDA Web Service client.
+
+        :type base_url: str
+        :param base_url: Base URL of FDSN/EIDA web service compatible server
+            (e.g. "http://geofon.gfz-potsdam.de") or key string for recognized
+            server. See :mod:`FDSN client documentation <obspy.clients.fdsn>`
+        :type debug: bool
+        :param debug: Debug flag.
+        :type timeout: float
+        :param timeout: Maximum time (in seconds) to wait for a single request
+            to receive the first byte of the response (after which an exception
+            is raised).
+        :type retry_count: int
+        :param retry_count: Number of retries.
+        :type retry_wait: int
+        :param retry_wait: Seconds to wait before each retry.
+        :type maxthreads: int
+        :param maxthreads: Maximum number of download threads.
+        :type credentials: dict
+        :param credentials: url => (username, password).
+        :type authdata: string
+        :param authdata: Authentication token (PGP base64 format).
+
+        """
+        super(Client, self).__init__(base_url, **kwargs)
+        self.__retry_count = retry_count
+        self.__retry_wait = retry_wait
+        self.__maxthreads = maxthreads
+        self.__credentials = credentials
+        self.__authdata = authdata
+
+    def _download(self, url, return_string=False, data=None, use_gzip=True):
+        u = urlparse.urlparse(url)
+        q = dict((p, v[0]) for (p, v) in urlparse.parse_qs(u.query).items())
+
+        if '/dataselect/' in u.path:
+            q['service'] = 'dataselect'
+
+        elif '/station/' in u.path:
+            q['service'] = 'station'
+
+        else:  # 'event' is not routed
+            return super(Client, self)._download(
+                    url, return_string, data, use_gzip)
```

This might potentially mislead users - I would natively expect that events are routed as well. IMHO it might be better to just raise an error here and tell people you use the normal fdsn client. Opinions?